python/rascaline/rascaline/splines.py

```diff
@@ -1,28 +1,29 @@
+from typing import Callable, Optional
+
 import numpy as np
 
 
 def generate_splines(
-    radial_basis,
-    radial_basis_derivatives,
-    max_radial,
-    max_angular,
-    cutoff_radius,
-    n_spline_points=None,
-    requested_accuracy=1e-8,
+    radial_basis: Callable,
+    radial_basis_derivatives: Callable,
+    max_radial: int,
+    max_angular: int,
+    cutoff_radius: float,
+    n_spline_points: Optional[int] = None,
+    requested_accuracy: Optional[float] = 1e-8,
```

Comment:
```suggestion
    requested_accuracy: float = 1e-8,
```

`Optional[T]` means `Union[None, T]`